t51-projects/gh-project-board.php

```diff
@@ -1,45 +1,70 @@
 <?php
 
 
-function curl_fetch() {
+$url = 'https://api.github.com/users/jessLundie'; // Set source URL
 
-	$url = 'https://api.github.com/users/jessLundie'; // Set source URL
+$media_header = array(
+	"Accept: application/vnd.github.inertia-preview+json",
+); // Custom media header required for projects in beta
 
-	$column_id = "15445284"; // Column id, found in the column link here: https://d.pr/i/W7RQJB
+$user_agent = "jessLundie"; // Set user agent
```

Comment:
we know from the code itself that we are setting the `user_agent`, but I'm left asking "why is `jessLundie` the User Agent string?" and "why are we setting this?"

the comment here could be a great opportunity to make that case. it could be too that from the outside looking in, we don't care at all that this is communicated to GitHub via the User Agent. what matters _from the outside_? that's the name we want to use _as our interface_.

possibly, `$username = "jessLundie";`

Reply:
Ah, I see. From the outside, I think what matters is that this value is set to the account where the project you're trying to access lives. In this case, it could be a GitHub username or an organization name, for an organization-owned project. 

In this case, maybe just:

`$user = "jessLundie"; // Project owner - GH username or organization`

Reply:
or even `$project_owner = "jessLundie"; // GH username or organization`